pkg/algorithm/cstorpoolselect/v1alpha1/select.go

```diff
@@ -16,6 +16,7 @@ type labelKey string
 const (
 	preferReplicaAntiAffinityLabel labelKey = "openebs.io/preferred-replica-anti-affinity"
 	replicaAntiAffinityLabel       labelKey = "openebs.io/replica-anti-affinity"
+	cstorPoolUIDLabelKey           labelKey = "cstorpool.openebs.io/uid"
```

Comment:
Naming convention is changed. Why is `Key` introduced as a suffix?

Reply:
yeah updated it